src/utils/resources/vm/utils/utils.ts

```diff
@@ -18,3 +19,6 @@ export const getVMListPathWithRowFilters = (
   namespace: string,
   rowFilters: Record<string, string>,
 ) => getVMListPath(namespace, getRowFiltersString(rowFilters));
+
+export const getVCPUCount = (cpu: V1CPU) =>
+  (cpu.sockets || 1) * (cpu.cores || 1) * (cpu.threads || 1);
```

Comment:
the method receives (most of the time) the output from `getCpu(vmi)` which may be undefined. Some cases are guarded with `isEmpty` or equivalent but some are not. We need either make this method return reasonable default data for undefined or verify each usage.